src/bundle/Resources/public/scss/_card.scss

```diff
@@ -34,14 +34,16 @@
             padding: .25rem 1.25rem;
             justify-content: space-between;
             z-index: 1040;
+        }
+
+        .form-inline {
+            .btn-danger {
+                display: flex;
+                height: calculateRem(48px);
 
-            .form-inline {
-                .btn-danger {
-                    .ez-icon {
-                        width: 1.5rem;
-                        height: 1.5rem;
-                        margin-top: 5px;
-                    }
+                .ez-icon-trash {
```

Comment:
You can use class `ez-icon--medium` to set 24px height and width

Reply:
Yes, but I have decided to add `.ez-icon-trash` instead, because in `create.html.twig` this statement:
https://github.com/ezsystems/ezplatform-admin-ui/blob/5b9dce453200d89961b0082c8ad0b747dd4f6e18/src/bundle/Resources/views/admin/content_type/create.html.twig#L64 
generates button with `.ez-icon-trash`. Also, other trash buttons in our system have `.ez-icon-trash` class. For consistency, I think we should use this class here as well.

Reply:
But adding class to set width and height is redundant. For having icons with width and height we have special modifier class `ez-icon--medium`. I'm not against adding `.ez-icon-trash`, I'm against using this class to set 24px width and height.

Reply:
You're right, better idea would be to use `.ez-icon--medium`. Also, I realised that the template for `{{ form_widget(form.removeFieldDefinition ...` is in the same file, so I can add `.ez-icon--medium` there. 
Unfortunately, after applying changes I realised that we are styling `.ez-card__header .form-inline .btn-danger .ez-icon` in `_icons.scss`:
https://github.com/ezsystems/ezplatform-admin-ui/blob/5b9dce453200d89961b0082c8ad0b747dd4f6e18/src/bundle/Resources/public/scss/_icons.scss#L76-L81
what overwrites `.ez-icon--medium` selector. Probably this is why someone before me styled it here using this selector: `.ez-card__header--sticky-top .form-inline .btn-danger .ez-icon` (I only changed `.ez-icon` -> `.ez-icon-trash` and moved out from `--sticky-top`).

So, I think that my current solution is valid in context of this task. Definitely, it would be great to refactor `.ez-card__header .form-inline .btn-danger .ez-icon` from `_icons.scss`, but I think that it should be done in another task/PR. 
What do you think, @dew326 ? 🙂

Reply:
Maybe you should fix that selector as well and apply a proper CSS class the elements defined by `.ez-card__header .form-inline .btn-danger .ez-icon` selector?